app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Welcome to the Driver List</h1>
+
+<ul>
```

Comment:
Tip: a [`<table>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) can make displaying this sort of information easier.